pkgs/data/fonts/intel-one-mono/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  src = fetchzip {
+    url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
```

Comment:
There's also an opentype font that should be installed. See #231784 for reference.
With the new 1.2.0 release I guess this could also use `fetchFromGitHub` since the tagged release now has `otf` & `ttf` folders.